airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/schemas/ads_insights.json

```diff
@@ -148,10 +144,6 @@
       "description": "Cost per conversion",
       "$ref": "ads_action_stats.json"
     },
-    "cost_per_conversion_lead": {
```

Comment:
Let's add `lead` field to `/schemas/shared/ads_action_stats.json`:
```json
"lead": {
   "type": ["null", "number"]
}
```

Reply:
Done